proto/convert_proto_to_swift.sh

```diff
@@ -2,10 +2,8 @@
 
 ## This script converts *.proto files to *.pb.swift files.
 ## In order to work properly protoc should be installed.
-cd resources
 
-set -euo pipefail
```

Comment:
Should be part of most scripts here. It is default in CircleCI but not when you run it locally

```suggestion
set -euo pipefail
```

further reading: https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
Ah thanks did not know about this 👍